src/components/AgGrid/TopicEntityTagActions.jsx

```diff
@@ -28,18 +52,38 @@ export default (props) => {
             if (response.status === 204) {
                 // remove the deleted item from the state so that the UI updates
                 props.api.applyTransaction({ remove: [ props.api.getRowNode(props.node.id).data ] });
+
             } else {
                 console.error("Failed to delete the item:", response.data);
             }
         } catch (error) {
             console.error("Error deleting item:", error);
         }
+        setShowModal(false);
+     }
     };
 
 
     return (
     <span>
-        {props.data.topic_entity_tag_source.secondary_data_provider_abbreviation === accessLevel ? <button onClick={() => handleDeleteClick()}>Delete</button> : null}
```

Comment:
The way that this used to work with a ? operator was a bit more explicit on what exactly is being shown.  Instead now the button is 10ish lines of code down.  Not sure if there is a reason for this.

Reply:
@pjhale I changed back to original way.